pkg/chunked/cache_linux.go

```diff
@@ -111,7 +111,7 @@ func getLayersCacheRef(store storage.Store) *layersCache {
 		cache.refs++
 		return cache
 	}
-	cache := &layersCache{
+	cache = &layersCache{
```

Comment:
(I wonder about making this kind of mistake harder to do … naming the global `cacheSingleton`, `globalCache`, or something, might make it less likely to conflict with a local. OTOH that’s very likely an overreaction.)